README.md

```diff
@@ -126,6 +126,14 @@ vars:
       - name: "new_custom_field"
         alias: "custom_field"
       - name: "a_second_field"
+    reddit_ads__account_conversions_passthrough_metrics:
```

Comment:
We should probably provide an example of how the alias would be applied.

Reply:
added!